.github/dependabot.yml

```diff
@@ -28,3 +28,9 @@ updates:
   open-pull-requests-limit: 3
   labels:
     - "vertical-pod-autoscaler"
+- package-ecosystem: gomod
+  directory: "/addon-resizer"
+  schedule:
+    interval: daily
+  target-branch: "addon-resizer-release-1.8"
```

Comment:
Why target branch?

Reply:
This branch is the one been active manintained for `addon-resizer`.